src/lib.rs

```diff
@@ -199,10 +229,14 @@ impl Election {
     fn purge_spoiled_votes(&mut self) -> u64 {
         let before_length = self.votes.len();
         let candidates = self.candidates.as_slice();
+        let logger = &self.logger;
         self.votes.retain(|vote| {
             for candidate in vote {
                 if !candidates.contains(candidate) {
-                    info!("Candidate voted for but not running: {}.", candidate);
+                    debug!(
```

Comment:
not related to this pr but should support for write ins be on the roadmap

Reply:
That'd be a nice feature for 0.3.